app-backend/datamodel/src/main/scala/StacExport.scala

```diff
@@ -122,7 +122,8 @@ object StacExport {
       name: String,
       license: StacExportLicense,
       taskStatuses: List[TaskStatus],
-      annotationProjectId: UUID
+      annotationProjectId: UUID,
+      campaignId: UUID
```

Comment:
I don't think this needs to be adjusted -- @alkamin should chime in here, but I think there are two types of exports:
 1. Project Exports
 2. Campaign Exports

For this PR I think the query parameter addition and adding the filtering is the important part because adding a filter for campaign id on the STAC export link allows us to filter for campaign exports.

I hope that cleans up some of the testing too.